src/api/app/views/webui/project/_form.html.haml

```diff
@@ -7,7 +7,7 @@
       .input-group-prepend
         %span.input-group-text
           #{namespace}:
-      = form.text_field(:name, required: true, class: 'form-control')
+      = form.text_field(:name, required: true, class: 'form-control', autofocus: true)
```

Comment:
This `autofocus: true` should also be on this line 14 since this text field is only displayed when there is a namespace (so for subprojects).

Reply:
Right!